backend/controllers/images.ts

```diff
@@ -38,6 +38,7 @@ export const imageFactory = (prisonApi) => {
       prisonApi
         .getPrisonerImage(res.locals, offenderNo, fullSizeImage)
         .then((data) => {
+          res.set('Cache-control', 'private, max-age=86400')
```

Comment:
set to be private so that only cached in browsers and not in intermediate servers.  86400 is one day